R/data.table.R

```diff
@@ -2505,28 +2495,16 @@ setnames = function(x,old,new,skip_absent=FALSE) {
 
 setcolorder = function(x, neworder=key(x))
 {
-  if (anyDuplicated(neworder)) stop("neworder contains duplicates")
```

Comment:
this was a different error right? `setcolorder(x, c(1, 1, 2, 3))`?

Reply:
Yes, many error messages got changed as they are now handled in single place

Reply:
I meant the test was for different behavior?

Reply:
these tests duplicates on `names(x)`, not `neworder` argument which is checked inside `colnamesInt`. Check for duplicates in `names(x)` was made later in `if (is.character(neworder))` branch.